nca/NetworkConfig/LiveSim/istio_gateway/istio_gateway.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: istio-ingressgateway-livesim
+  labels:
+    app: istio-ingressgateway
+    istio: ingressgateway
+spec:
+  containers:
```

```suggestion
  serviceAccountName: istio-ingressgateway
  containers:
```
add `serviceAccountName `, sometimes it is referred by authorization policies

---

done